pandas/tests/frame/methods/test_sort_index.py

```diff
@@ -441,22 +441,22 @@ def test_sort_index_ignore_index(
                 {"M1": [1, 2], "M2": [3, 4]},
                 True,
                 False,
-                MultiIndex.from_tuples([[2, 1], [3, 4]], names=list("AB")),
+                MultiIndex.from_tuples(((2, 1), (3, 4)), names=list("AB")),
             ),
             (
                 {"M1": [1, 2], "M2": [3, 4]},
                 {"M1": [2, 1], "M2": [4, 3]},
                 False,
                 False,
-                MultiIndex.from_tuples([[3, 4], [2, 1]], names=list("AB")),
+                MultiIndex.from_tuples(((3, 4), (2, 1)), names=list("AB")),
             ),
         ],
     )
     def test_sort_index_ignore_index_multi_index(
         self, inplace, original_dict, sorted_dict, ascending, ignore_index, output_index
     ):
         # GH 30114, this is to test ignore_index on MulitIndex of index
-        mi = MultiIndex.from_tuples([[2, 1], [3, 4]], names=list("AB"))
+        mi = MultiIndex.from_tuples(((2, 1), (3, 4)), names=list("AB"))
```

Comment:
this should still be a list of tuples (same as above); yes this works, but its a test so should change.

Reply:
done